src/pip/_vendor/certifi/core.py

```diff
@@ -1,15 +1,114 @@
-# -*- coding: utf-8 -*-
-
 """
 certifi.py
 ~~~~~~~~~~
 
-This module returns the installation location of cacert.pem.
+This module returns the installation location of cacert.pem or its contents.
 """
-import os
+import sys
+import atexit
+
+def exit_cacert_ctx() -> None:
```

Comment:
Ok, just on a cursory pass at the patch itself (I haven't verified provenance, etc yet - the scary stuff from my POV / assuming the worst to put safety 1st) - this won't work with the Python3.5+ type annotations since Pex supports `>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*<3.14` and this would fail under Python 2.7.

So, at present, the patch will need to be applied to a different branch and it will need to at least have an edit to remove all type annotations.